src/flows/data_retention/development_snowflake_schemas.py

```diff
@@ -0,0 +1,49 @@
+"""DELETE all DEVELOPMENT DB schemas older than 89 days
+
+Reference: https://getpocket.atlassian.net/wiki/spaces/CP/pages/2703949851/Snowflake+Data+Deletion+and+Retention
+"""
+from prefect import Flow, task, unmapped
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery
+from utils.flow import get_flow_name
+
+FLOW_NAME = get_flow_name(__file__)
+DATABASE = "DEVELOPMENT"
+
+# Run daily at midnight UTC.
+SCHEDULE = Schedule(clocks=[CronClock("0 0 * * *")])
+
+GET_SCHEMAS_SQL = """
+SELECT catalog_name as database,
+       schema_name,
+       schema_owner,
+       created,
+       last_altered
+FROM information_schema.schemata
+WHERE created < DATEADD("day", -89, CURRENT_TIMESTAMP())
+AND schema_name != 'DBT_SBISWAS';
+"""
+"""
+We are excluding schema `DBT_SBISWAS` as its being actively used for delivering Android push notifications experiments.
+Kirill will be doing an experiment to explore doing this work our DBT workflow and as a result putting this data in our
```

Comment:
@sbiswas27 and I did some brainstorming here and I'm fairly confident that in the future we can dbt models so they are refreshed daily, but their result set does not change (i.e. by hard-coding dates and not allowing offline events to become part of said models).

Now, on the subject of the specific `development.dbt_sbiswas.PUSH_NOTIFICATIONS_TWAU_TEST` table, we have an issue in the near term. There were some issues setting this table up that mean we cannot 100% replicate the AB test assignments in it using a dbt model written from scratch. We will want to keep this table around for the next month or two while we document the findings of the TWAU Push Notification test. After that timeline, it can be dropped.

Reply:
@kirill-demtchouk We're clear of this and can move forward with this PR, right?

Reply:
@jeshuaborges Based on Kirill's response, I think you're unblocked on this PR, and you can remove `DBT_SBISWAS` from the list of excluded `schema_name`.